### ISSUE TITLE
File upload

### ISSUE THREAD

**Issue by VRDD** [schorschii/webpw#3]
Hi,
thanks for this software!
Would it be possible to include the possibility of fileuploads in addition to passwords, comments and so on?
Thanks for considering this request.
Cheers Veit

**Comment by schorschii** [comment]
Yes, file upload support is already on the road map but not planned for the 1.0.0 release.
